package.nls.json

```diff
@@ -50,6 +50,8 @@
   "configuration.SAS.flowConversionMode": "Choose the conversion mode for notebooks",
   "configuration.SAS.flowConversionModeNode": "Convert each notebook cell to a node",
   "configuration.SAS.flowConversionModeSwimlane": "Convert each notebook cell to a swimlane",
+  "configuration.SAS.logFocus.onExecutionFinish": "Focus SAS Log when Code Execution is finished",
```

Comment:
Suggest to say "show" instead of "focus". It does not focus as my [comment](https://github.com/sassoftware/vscode-sas-extension/issues/647#issuecomment-1829154110) explained.
Also, I'm not sure why "Code Execution" needs uppercase.

Reply:
See renames in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/0ce419ea20fce295cf620a7f6b83f9a0c0ff9104. Are those names sufficient or should we explore different ones?